pcli/src/command/validator.rs

```diff
@@ -1,24 +1,45 @@
+use std::fs::File;
+
+use anyhow::Context;
 use anyhow::Result;
-use penumbra_stake::IdentityKey;
-use penumbra_wallet::ClientState;
+use penumbra_stake::Validator;
+use penumbra_stake::ValidatorDefinition;
+use rand_core::OsRng;
 use structopt::StructOpt;
 
+use penumbra_proto::{stake::Validator as ProtoValidator, Message};
```

Comment:
I think a slightly cleaner pattern here would be:
```
use penumbra_proto::{stake as pb, Message};
```
so that the protobuf type can be referred to as `pb::Validator`.

Reply:
(this is non-blocking, though!)